run.sh

```diff
@@ -0,0 +1,38 @@
+
+if [[ "$1" == "antwerp" ]]; then
+
+    module use /apps/antwerpen/modules/centos8/software-admin-x86_64
+    module load ReFrame/3.9.1
+
+    export RFM_CONFIG_FILE=$(dirname $0)/config_vsc.py
```

Comment:
`RFM_CONFIG_FILE` is the same for each site, so you can move it out of the if block.

Reply:
reasonable